src/main/resources/mixins.hodgepodge.late.json

```diff
@@ -4,6 +4,9 @@
   "package": "com.mitchej123.hodgepodge.mixins.late",
   "refmap": "mixins.hodgepodge.refmap.json",
   "target": "@env(DEFAULT)",
-  "compatibilityLevel": "JAVA_8"
+  "compatibilityLevel": "JAVA_8",
+  "mixins": [
+    "extrautilities.MixinFluidBufferRetrieval"
+  ]
```

Comment:
There's no need to manually register mixin classes. These entries are dynamically generates from classes listed in https://github.com/GTNewHorizons/Hodgepodge/blob/master/src/main/java/com/mitchej123/hodgepodge/mixins/Mixins.java

Reply:
Makes total sense. Mixins.java was one of the last things I found, embarrassingly, so I had already registered it manually. Thanks!

Reply:
Then please, revert changes in JSON file (remove these unnecessary lines).

Reply:
Done, sorry about that